app/src/main/java/fr/free/nrw/commons/delete/DeleteHelper.java

```diff
@@ -156,16 +158,24 @@ public void askReasonAndExecute(Media media,
         ArrayList<Integer> mUserReason = new ArrayList<>();
 
         String[] reasonList = {"Reason 1", "Reason 2", "Reason 3"};
+        String[] reasonListEnglish = {"Eng1", "Eng2", "Eng3"};
```

Comment:
Please add comment: `// Messages posted on-wiki should not be in the app user's locale, but rather in Commons' lingua franca English.`